src/components/containers/PostListContainer.js

```diff
@@ -9,27 +9,26 @@ class PostListContainer extends Component {
     componentDidMount() {
         const { dispatch } = this.props;
 
-        dispatch(getPosts('http://localhost:9000/posts.json'));
+        dispatch(getPosts('/posts.json'));
     };
 
     render() {
-        const { hasError, isFetching, posts } = this.props;
+        const { status, posts } = this.props;
 
         return (
-            <div>
-                { hasError && <p>There was an error loading the items</p> }
-                { isFetching && <PreLoader /> }
+            <section>
+                { (status == 'error') && <p>There was an error loading the items</p> }
```

Comment:
May be better to move content definition in separated method, because currently here just one tag for each 'state', but later it could be more than one